<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
  background-color: #fff;
  border: 1px solid #dbdbdb;
  border-radius: .375em;
  color: #363636;
  font-family: &quot;Segoe UI&quot;;
  font-size: 1rem;
  height: 2.5em;
  line-height: 1.5;
  padding: calc(0.5em - 1px) 1em;
  position: relative;
  text-align: center;
  vertical-align: top;
  white-space: nowrap;
}

QPushButton::hover{
  border-color: #5500ff;
  outline: 0;
}

QPushButton::pressed {
  border-color: #4a4a4a;
  outline: 0;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>271</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>111</width>
      <height>102</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="comboBox_type">
       <item>
        <property name="text">
         <string>Simple Maze</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Middle Maze</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Complex Maze</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_form">
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Rectangular</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Circular</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Triangular</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hexagonal</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_rect_type">
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Orthogonal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sigma</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Delta</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_size">
       <item>
        <property name="text">
         <string>Minimal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Middle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Large</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
